lib/facebook/graph_api.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Facebook.GraphAPI do
+  @moduledoc false
+
+  use HTTPoison.Base
+
+  alias Facebook.Config
+
+  def process_url(url), do: Config.graph_url <> url
```

Comment:
I have a pattern matching idea for how to combine this with the `GraphVideoApi` module, I just need a sec to test it. Essentially I think we can pattern match off a binary passed in